### DISCUSSION TITLE
Adding TelephoneNumber and EmailAddress

### DISCUSSION THREAD

**Discussion by rockc15** [CommonCoreOntology/CommonCoreOntologies#400] [Announcements]
We can represent both as a DesignativeICE that designates a telecommunications endpoint assigned to an EmailBox or Telephone. However, from a user's perspective, the terms are commonly encountered and the semantics are awkward. Having the terms would help outside developers and users.

**Comment by APCox** [comment]
Both terms already exist in an extension ontology (Agent History) as subclasses of 'Telecommunication Network Identifier', which is a DesignativeICE. They should be reviewed for promotion to the mid-level.

**Comment by neilotte** [comment]
@APCox There are now at least two issues that have been moved into discussion that hang on adding patterns around PII and ensuring they are robust in CCO and satisfy competency questions. I'm moving this one over there as well so that these issues may be aggregates and addressed in a workplan.
